dbms/tests/integration/helpers/docker_compose_kafka.yml

```diff
@@ -15,14 +15,17 @@ services:
     image: confluentinc/cp-kafka:4.1.0
     hostname: kafka1
     ports:
-      - "9092:9092"
+        - "9092:9092"
     environment:
-      KAFKA_ADVERTISED_LISTENERS: "PLAINTEXT://kafka1:9092"
-      KAFKA_ZOOKEEPER_CONNECT: "kafka_zookeeper:2181"
-      KAFKA_BROKER_ID: 1
-      KAFKA_LOG4J_LOGGERS: "kafka.controller=INFO,kafka.producer.async.DefaultEventHandler=INFO,state.change.logger=INFO"
-      KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR: 1
+        KAFKA_ADVERTISED_LISTENERS: INSIDE://localhost:9092,OUTSIDE://kafka1:19092
```

Comment:
These changes were made to allow the python test `test_kafka_protobuf` to publish messages to kafka.